DataProcessor/src/net/relinc/processor/controllers/TrimDataController.java

```diff
@@ -564,6 +558,7 @@ public void removeModifierButtonFired(){
 	
 	@FXML
 	public void applyModifierButtonFired(){
+		getActivatedData().invalidateResult();
```

Comment:
The result of running modifiers on the data is cached. This empties that cache. So when we change a modifier, we need to empty the cache. With all the side-effect programming, there's not a sweet way to do it with more confidence